osquery/core/init.cpp

```diff
@@ -122,6 +122,16 @@ CLI_FLAG(uint64,
          alarm_timeout,
          15,
          "Seconds to allow for shutdown. Minimum is 10");
+#ifdef OSQUERY_LINUX
+
+/* The default value here is just a placeholder,
+   it will be recalculated at runtime */
```

Comment:
Well, I need to define a flag and it doesn't support a non-initialized value; so that's the definition, but given that its value it's dynamic if not provided by the user, then the value is whatever the runtime calculates with the rules described in the full description of the flag.